.github/workflows/validation.yml

```diff
@@ -99,6 +99,34 @@ jobs:
           cd framework/pipeline/fileopening
           restManager PrintTrees v2.2.30_hits.root
 
+  macros-health:
+    name: "Macros with clean error output"
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/lobis/root-geant4-garfield:rest-for-physics
+    needs: [ framework-install ]
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          repository: rest-for-physics/framework
+          path: framework
+      - name: Restore cache
+        uses: actions/cache@v3
+        id: framework-install-cache
+        with:
+          key: ${{ env.BRANCH_NAME }}-${{ github.sha }}
+          path: ${{ env.REST_PATH }}
+      - name: Load restRootMacros
+        run: |
+          source ${{ env.REST_PATH }}/thisREST.sh
+          cd framework/pipeline
+          export DISPLAY=localhost:0.0
+          echo $GARFIELD_INSTALL
+          ls $GARFIELD_INSTALL
+          export ROOT_INCLUDE_PATH=$ROOT_INCLUDE_PATH:/usr/local/garfieldpp/install/Include
```

Comment:
`ROOT_INCLUDE_PATH` already points to the Garfield install on the docker image, you can check this on the image itself looking at the environment variables.

Reply:
For quick reference here are the env variables. I used `/bin/sh` as the shell to avoid loading variables from the .bashrc so I think this should be the exact same variables active when running the pipeline in a non-interactive shell:

```
# docker run -it --rm ghcr.io/lobis/root-geant4-garfield-dev:rest-for-physics /bin/sh
# printenv
GARFIELD_INSTALL=/usr/local/garfieldpp/install
G4INCLDATA=/usr/local/geant4/install/share/Geant4-11.0.3/data/G4INCL1.0
HOSTNAME=6d130f6981a5
LD_LIBRARY_PATH=/usr/local/geant4/install/lib:/usr/local/garfieldpp/install/lib:/usr/local/root/install/lib:
HOME=/root
G4PARTICLEXSDATA=/usr/local/geant4/install/share/Geant4-11.0.3/data/G4PARTICLEXS4.0
G4LEDATA=/usr/local/geant4/install/share/Geant4-11.0.3/data/G4EMLOW8.0
G4LEVELGAMMADATA=/usr/local/geant4/install/share/Geant4-11.0.3/data/PhotonEvaporation5.7
G4RADIOACTIVEDATA=/usr/local/geant4/install/share/Geant4-11.0.3/data/RadioactiveDecay5.6
G4PIIDATA=/usr/local/geant4/install/share/Geant4-11.0.3/data/G4PII1.3
G4SAIDXSDATA=/usr/local/geant4/install/share/Geant4-11.0.3/data/G4SAIDDATA2.0
G4REALSURFACEDATA=/usr/local/geant4/install/share/Geant4-11.0.3/data/RealSurface2.2
GARFIELD_VERSION=eb1b5403
GEANT4_VERSION=v11.0.3
TERM=xterm
ROOT_VERSION=v6-26-06
ROOTSYS=/usr/local/root/install
G4ABLADATA=/usr/local/geant4/install/share/Geant4-11.0.3/data/G4ABLA3.1
PATH=/usr/local/geant4/install/bin:/usr/local/root/install/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
HEED_DATABASE=/usr/local/garfieldpp/install/share/Heed/database
G4NEUTRONHPDATA=/usr/local/geant4/install/share/Geant4-11.0.3/data/G4NDL4.6
DEBIAN_FRONTEND=noninteractive
ROOT_INCLUDE_PATH=/usr/local/garfieldpp/install/include/Garfield:/usr/local/garfieldpp/install/include:
G4ENSDFSTATEDATA=/usr/local/geant4/install/share/Geant4-11.0.3/data/G4ENSDFSTATE2.3
CMAKE_CXX_STANDARD=17
PWD=/
PYTHONPATH=/usr/local/root/install/lib:
TINI_VERSION=v0.19.0
CMAKE_PREFIX_PATH=/usr/local/garfieldpp/install:
```

Reply:
However, when you `source thisREST.sh` it will be overwritten by the `ROOT_INCLUDE_PATH` definition found there.

Reply:
If that was the problem, the next commit should solve it.

Reply:
It must be `thisREST.sh` who sets that up, so that the user does not need to care about setting any env variables